FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice/covg_multiple_priorities_no_timeslice_utest.c

```diff
@@ -1742,6 +1742,176 @@ void test_coverage_prvDeleteTCB_static_stack_only(void)
     UnityMalloc_EndTest();
 }
 
+/** @brief xTaskResumeFromISR - resume task from within ISR context
+ *
+ * <b>Coverage</b>
+ * @code{c}
+ *      if( prvTaskIsTaskSuspended( pxTCB ) != pdFALSE )
+ *      {
+ *          traceTASK_RESUME_FROM_ISR( pxTCB );
+ *          ...
+ *          if( uxSchedulerSuspended == ( UBaseType_t ) pdFALSE )
+ *          {
+ *              ...
+ *              prvAddTaskToReadyList( pxTCB );
+ * @endcode
+ *
+ * Cover the case where the scheduler is not suspended, and the
+ * task being resumed is suspended.
+ *
+ */
+void test_coverage_xTaskResumeFromISR_task_suspended_uxpriority_greater(void)
+{
+    TCB_t xTaskTCBs[ 2 ] = { NULL };
+    BaseType_t xAlreadyYielded;
+    UBaseType_t uxPriority, uxCore;
+    List_t xList;
+
+    for( uxPriority = (UBaseType_t)0U; uxPriority < (UBaseType_t)configMAX_PRIORITIES; uxPriority++ )
+    {
+        vListInitialise(&(pxReadyTasksLists[uxPriority]));
+    }
```

Comment:
can we move this code to a common section, as it is repeating multiple times, and not essential to the testcase understanding